devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -206,7 +206,8 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 			case 0x09: //TAMPER
 				sendEvent(name: "tamper", value: "detected")
 				sendEvent(name: "alarm", value: "both")
-				runIn(2, "clearTamperAndAlarm")
+				sendEvent(name: "chime", value: "chime")
```

Comment:
Do we actually want to send `chime` here? It should be safe to still clear it with `clearTamperAlarmAndChime`.

Reply:
I assume we want to send chime wherever we're sending alarm?

Reply:
@greens Yes, in my opinion, this is correct.

Reply:
But a chime isn't an alarm.

I also don't see how this resolves the issue reported in the ticket.